tagupy/design/analyzer/_met.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+from tagupy.type import _Analyzer as Analyzer
+from typing import NamedTuple
+
+
+class METResult(NamedTuple):
+    exmatrix: np.ndarray
+    resmatrix: np.ndarray
+    effectmatrix: np.ndarray
+
+
+class MET(Analyzer):
+    '''
+    Analyzer Class of main effect table (MET)
+
+    Notes
+    ----------
+    This analyzer provides analyzing main effect table.
+    You need two matrices: `exmatrix` and `resmatrix`.
+    The `exmatrix` represents which factors to include or not.
+    The `resmatrix` contains the results obtained by the experiment.
+
+    When you execute this analyzer, it return the analysis result created by `NamedTuple`.
+    This result includes three contents: `exmatrix`, `resmatris`, and `effectmatrix`.
+    `exmatrix` and `resmatrix` are same as each own input when execute analyze method.
+    `effectmatrix` is the result of analyzing the main effects for two inputs.
+
```

Comment:
`exmatrix` and `resmatrix` are **"the"** same as each own input when execute analyze method.
`effectmatrix` is the main effects of each factor, obtained through the analysis based on the two input matrices.

Reply:
thx